src/System.Private.Uri/src/System.Private.Uri.builds

```diff
@@ -10,14 +10,6 @@
     </Project>
     <Project Include="System.Private.Uri.csproj">
       <OSGroup>Windows_NT</OSGroup>
-      <TargetGroup>netstandard13aot</TargetGroup>
-    </Project>
-    <Project Include="System.Private.Uri.csproj">
```

Comment:
I am not seeing System.Private.Uri package today contains netstandard1.3aot so I am not sure if there is anyone today directly depend on the library building. @schellap was the one working on creating and getting the needed packages for corert so he may tell if we depend on that. 
But in general I believe we need to have netstandard1.7aot because even if corert not using it today, will need to use it. so I suggest we keep netstandard1.3aot for now till we decide what to do here.

---
In reply to: [80354849](https://github.com/dotnet/corefx/pull/11988#discussion_r80354849) [](ancestors = 80354849)